packages/crashlytics/ios/RNFBCrashlytics/RNFBCrashlyticsInitProvider.m

```diff
@@ -21,37 +21,54 @@
 #import "RNFBPreferences.h"
 #import "RNFBJSON.h"
 #import "RNFBMeta.h"
+#import "RNFBApp/RNFBSharedUtils.h"
 
 NSString *const KEY_CRASHLYTICS_DEBUG_ENABLED = @"crashlytics_debug_enabled";
 NSString *const KEY_CRASHLYTICS_AUTO_COLLECTION_ENABLED = @"crashlytics_auto_collection_enabled";
 
 @implementation RNFBCrashlyticsInitProvider
 
 + (void)load {
-  if ([self isCrashlyticsCollectionEnabled]) {
-    [FIRApp registerInternalLibrary:self withName:@"react-native-firebase-crashlytics" withVersion:@"6.0.0"];
-  }
+  [FIRApp registerInternalLibrary:self withName:@"react-native-firebase-crashlytics" withVersion:@"6.0.0"];
 }
 
 + (BOOL)isCrashlyticsCollectionEnabled {
   BOOL enabled;
 
   if ([[RNFBPreferences shared] contains:KEY_CRASHLYTICS_AUTO_COLLECTION_ENABLED]) {
     enabled = [[RNFBPreferences shared] getBooleanValue:KEY_CRASHLYTICS_AUTO_COLLECTION_ENABLED defaultValue:YES];
+    DLog(@"isCrashlyticsCollectionEnabled via RNFBPreferences: %d", enabled);
   } else if ([[RNFBJSON shared] contains:KEY_CRASHLYTICS_AUTO_COLLECTION_ENABLED]) {
     enabled = [[RNFBJSON shared] getBooleanValue:KEY_CRASHLYTICS_AUTO_COLLECTION_ENABLED defaultValue:YES];
+    DLog(@"isCrashlyticsCollectionEnabled via RNFBJSON: %d", enabled);
```

Comment:
These logs are a bit loud, admittedly, but I think it helps debugging enough that it may be worth it. Of course I'll defer to whatever the team wants!

Reply:
For something as vital as this, and only done once on startup, I personally like the logs.